packages/vuetify/src/components/VSheet/VSheet.ts

```diff
@@ -1,70 +1,101 @@
+import propsFactory from '@/util/propsFactory'
 
-// Helpers
-import mixins from '../../util/mixins'
+import {
+  makeBorderRadiusProps,
+  makeDimensionProps,
+  makeElevationProps,
+  makeTagProps,
+  useBorderRadius,
+  useDimension,
+  useElevation,
+  useTheme,
+} from '@/composables'
 
 // Types
-import { VNode } from 'vue'
-
-/* @vue/component */
-export default mixins(
-  BindsAttrs,
-  Colorable,
-  Elevatable,
-  Measurable,
-  Roundable,
-  Themeable
-).extend({
-  name: 'v-sheet',
-
-  props: {
-    outlined: Boolean,
-    shaped: Boolean,
-    tag: {
-      type: String,
-      default: 'div',
-    },
-  },
+export interface PositionProps {
+  absolute?: boolean
+  bottom?: boolean | string
+  fixed?: boolean
+  left?: boolean | string
+  right?: boolean | string
+  top?: boolean | string
+}
 
-  computed: {
-    classes (): object {
-      return {
-        'v-sheet': true,
-        'v-sheet--outlined': this.outlined,
-        'v-sheet--shaped': this.shaped,
-        ...this.themeClasses,
-        ...this.elevationClasses,
-        ...this.roundedClasses,
-      }
-    },
-    styles (): object {
-      return this.measurableStyles
-    },
-  },
+// Composables
+const makePositionProps = propsFactory({
+  absolute: Boolean,
+  bottom: [Boolean, String],
+  fixed: Boolean,
+  left: [Boolean, String],
+  right: [Boolean, String],
+  top: [Boolean, String],
+})
+
+function usePosition (props: PositionProps) {
+  const positionStyles = computed(() => {
+    const targets = ['top', 'right', 'bottom', 'left'] as const
+    const styles: Partial<Record<typeof targets[number] | 'position', string>> = {}
+    const pos = props.fixed ? 'fixed' : props.absolute ? 'absolute' : undefined
+
+    if (pos) styles.position = pos
+
+    for (const target of targets) {
+      const prop = props[target]
+
+      if (prop == null || prop === false) continue
 
-  render (h): VNode {
-    const data = {
-      class: this.classes,
-      style: this.styles,
-      on: this.listeners$,
+      styles[target] = convertToUnit(prop === true ? '0' : String(prop)) ?? '0'
     }
 
-    return h(
-      this.tag,
-      this.setBackgroundColor(this.color, data),
-      this.$slots.default
+    return styles
+  })
+
+  return { positionStyles }
+}
+
+const makeOutlineProps = propsFactory({
+  outlined: [Boolean, String],
+})
+
+export default defineComponent({
+  name: 'VSheet',
+
+  props: makeProps({
+    ...makeBorderRadiusProps(),
+    ...makeDimensionProps(),
+    ...makeElevationProps(),
+    ...makeOutlineProps(),
+    ...makePositionProps(),
+    ...makeTagProps(),
+  }),
+
+  setup (props, { attrs, slots }) {
+    const { borderRadiusClasses } = useBorderRadius(props)
+    const { elevationClasses } = useElevation(props)
+    const { themeClasses } = useTheme()
+    const { dimensionStyles } = useDimension(props)
+    const { positionStyles } = usePosition(props)
+
+    return () => (
+      h(props.tag, mergeProps(attrs, {
```

Comment:
Not sure if `mergeProps(attrs, ...)` is necessary here. V3 still automatically merges stuff when there is a single root afaik.